src/admin-views/integrations/plugins/elementor/template-selection-helper.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * Template for the Elementor template selection helper.
+ *
+ * This template is used to inject a helper message to the Elementor editor to help users understand that they are editing,
+ * the JavaScript and CSS are inline to avoid having the loading complexity for Elementor Edit area.
+ *
+ * @since TBD
+ *
+ * @package TEC\Events\Integrations\Plugins\Elementor\Template
+ */
+
+?>
+<style>
```

Comment:
Why add styles? 
For an injected element they could as easily be inline.
The styles are on the page even if the element doesn't get injected.

Reply:
True I will do that.

Reply:
Ahh it makes the JS funky. Lets keep it that way for now.